transaction/extra/src/memo/mod.rs

```diff
@@ -48,7 +48,7 @@
 //! | 0x0204          | Destination With Payment Intent Id Memo           |
 
 pub use self::{
-    authenticated_common::{compute_authenticated_sender_memo, compute_category1_hmac},
+    authenticated_common::compute_authenticated_sender_memo,
     authenticated_sender::AuthenticatedSenderMemo,
```

Comment:
This was never fully exposed publicly, newer versions of rust will throw a warning for unused code here.
With there being an implementation in `mc-crypto-memo-mac` it seemed best to consolidate to one implementation.